api/database/factories/UserFactory.php

```diff
@@ -209,4 +209,29 @@ public function configure()
             );
         });
     }
+
+    /**
+     * Attach roles to a user after creation.
+     *
+     * @param array $roles Array of the roles to attach
+     * @param string $team Name of the team to attach the roles to
+     *
+     * @return $this
+     *
+     * Usage:
+     *      `User::factory()->withRoles(['base_user', 'applicant']);`
+     *      `User::factory()->withRoles(['poo_operator'], 'team_name');`
```

Comment:
lol :hankey:

Reply:
oooopsss 😅 Fixed in [fix typo](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6663/commits/3a0e583a83aa8cf9b217a338f4df25eca3a34325)